### DISCUSSION TITLE
future data for wflow model

### DISCUSSION THREAD

**Discussion by mullerrobert799** [Deltares/hydromt#385] [Q&A]
Hello,

I am going to simulate the future period and have temperature and precipitation files of continuous data for future period. But I don't know how to get the radiation and sea level pressure data for future years. Could you please advise me on that? 

Do you have any provision directly in WFLOW/hydroMT where we can either get the future data or maybe make it by using some script or function?

**Comment by hboisgon** [comment]
Hi @mullerrobert799.
Well if your future data does not include radiation and sea level pressure then I would say it's a little bit up to you on how you would like to proceed. You could make an assumption and keep radiation and sea level pressure as the same as the historical data or decide to use long term monthly/daily/average based on 10-30 years of historical data. Or decide to switch to another dataset like CMIP or ISIMIP which also include these variables in their climate simulations.

So because it's up to you which assumption you want to make, we don't really have defaults in HydroMT then.
Hope this helps you out?
